pom.xml

```diff
@@ -223,6 +227,9 @@
                   <tags>
                     <tag>latest</tag>
                   </tags>
+                  <tags>
+                    <tag>${docker.image.additional.tag}</tag>
```

Comment:
this should be placed below the `latest` tag definition

Reply:
Done